xonsh/aliases.py

```diff
@@ -59,8 +61,9 @@ def EXEC_ALIAS_RE():
 class FuncAlias:
     """Provides a callable alias for xonsh commands."""
 
-    attributes_show = ["__xonsh_threadable__", "__xonsh_capturable__"]
+    attributes_show = ["__xonsh_threadable__", "__xonsh_capturable__", "return_command"]
     attributes_inherit = attributes_show + ["__doc__"]
+    return_command = False
```

Comment:
I try to avoid using boolean flags. They're pretty limiting and under-descriptive. In this case, for example, the reader knows that `return_command = True` means to return the command, but they might be left wondering, "what is returned if `return_command = False`?"

Consider instead:

```
return_what : Literal["command", "result"] = "result"
```

This approach is more descriptive, moves the selector (command or not) out of the variable name and into the value, and is more extensible should a new use-case arise for another variant of what to return.